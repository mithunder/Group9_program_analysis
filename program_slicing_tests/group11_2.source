
module Two:
read x;
read y;
read z;
if x=y -> write x
[] x=z -> write x
[]@programslicing="true"
	y=z -> write y
fi
end

/*
Original expectation: The read y and read z statements are returned.
*/
