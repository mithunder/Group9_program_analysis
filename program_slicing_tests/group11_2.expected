
module Two:
read x;
read y;
read z;
if x=y -> skip
[] x=z -> skip
[]@programslicing="true"
   y=z -> skip
fi
end

/*
Original expectation: The read y and read z statements are returned.
*/
